packages/plugin-ext/src/plugin/workspace.ts

```diff
@@ -50,6 +50,7 @@ export class WorkspaceExtImpl implements WorkspaceExt {
     public readonly onDidChangeWorkspaceFolders: Event<theia.WorkspaceFoldersChangeEvent> = this.workspaceFoldersChangedEmitter.event;
 
     private folders: theia.WorkspaceFolder[] | undefined;
+    private workSpaceFile: theia.Uri | undefined;
```

Comment:
`workSpaceFile` -> `workspaceFile`

Reply:
workspaceFile will collide with the property get, what about _workspaceFile, i.e. using a leading underscore character